x/ibc/02-client/client/cli/query.go

```diff
@@ -88,85 +88,11 @@ $ %s query ibc client state [client-id]
 // the chain as defined in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#query
 func GetCmdQueryConsensusState(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "consensus-state [client-id]",
-		Short:   "Query the latest consensus state of the client",
-		Long:    "Query the consensus state for a particular light client",
-		Example: fmt.Sprintf("%s query ibc client consensus-state [client-id]", version.ClientName),
-		Args:    cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
-			clientID := args[0]
-			if strings.TrimSpace(clientID) == "" {
-				return errors.New("client ID can't be blank")
-			}
-
-			prove := viper.GetBool(flags.FlagProve)
-
-			csRes, err := utils.QueryConsensusState(cliCtx, clientID, prove)
-			if err != nil {
-				return err
-			}
-
-			return cliCtx.PrintOutput(csRes)
-		},
-	}
-	cmd.Flags().Bool(flags.FlagProve, true, "show proofs for the query results")
-	return cmd
-}
-
-// GetCmdQueryRoot defines the command to query a verified commitment root
-func GetCmdQueryRoot(queryRoute string, cdc *codec.Codec) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "root [client-id] [height]",
-		Short: "Query a verified commitment root",
-		Long: strings.TrimSpace(
-			fmt.Sprintf(`Query an already verified commitment root at a specific height for a particular client
-
-Example:
-$ %s query ibc client root [client-id] [height]
-`, version.ClientName),
-		),
-		Args: cobra.ExactArgs(2),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
-			clientID := args[0]
-			if strings.TrimSpace(clientID) == "" {
-				return errors.New("client ID can't be blank")
-			}
-
-			height, err := strconv.ParseUint(args[1], 10, 64)
-			if err != nil {
-				return fmt.Errorf("expected integer height, got: %v", args[1])
-			}
-
-			prove := viper.GetBool(flags.FlagProve)
-
-			rootRes, err := utils.QueryCommitmentRoot(cliCtx, clientID, height, prove)
-			if err != nil {
-				return err
-			}
-
-			return cliCtx.PrintOutput(rootRes)
-		},
-	}
-	cmd.Flags().Bool(flags.FlagProve, true, "show proofs for the query results")
-	return cmd
-}
-
-// GetCmdQueryCommitter defines the command to query the committer of the chain
-// at a given height
-func GetCmdQueryCommitter(queryRoute string, cdc *codec.Codec) *cobra.Command {
```

Comment:
And this isn't either?

Reply:
not required anymore as the spec was updated to include it